src/main/java/com/powsybl/network/hypothesis/GeneratorStartup.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.hypothesis;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Generator;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface GeneratorStartup extends Extension<Generator> {
+
```

Comment:
It seems that we are missing the start up cost of the generator, when its `P` is equal to zero.

Reply:
I added a field to get this information, is it okay for you?